### ISSUE TITLE
help@reframedb.org blocked by scripps anti-spoofing (?)

### ISSUE THREAD

**Issue by flaneuse** [sebotic/repurpos.us#122]
This is the mail system at host eforward3b.registrar-servers.com.

I'm sorry to have to inform you that your message could not
be delivered to one or more recipients. It's attached below.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                  The mail system

<lhughes@scripps.edu> (expanded from <help@reframedb.org>): host
   us-smtp-inbound-1.mimecast.com[205.139.110.181] said: 550 Rejected by
   header based Anti-Spoofing policy: slelong@scripps.edu -
   https://community.mimecast.com/docs/DOC-1369#550 (in reply to end of DATA
   command)
Reporting-MTA: dns; eforward3b.registrar-servers.com
X-Postfix-Queue-ID: 27C9C140276
X-Postfix-Sender: rfc822; SRS0+gxXZ=LH=scripps.edu=slelong@eforward3b.registrar-servers.com
Arrival-Date: Fri, 24 Aug 2018 18:15:28 -0400 (EDT)

Final-Recipient: rfc822; lhughes@scripps.edu
Original-Recipient: rfc822;help@reframedb.org
Action: failed
Status: 5.0.0
Remote-MTA: dns; us-smtp-inbound-1.mimecast.com
Diagnostic-Code: smtp; 550 Rejected by header based Anti-Spoofing policy:
   slelong@scripps.edu - https://community.mimecast.com/docs/DOC-1369#550

**Comment by andrewsu** [comment]
I'm looking at this issue now -- trying to figure out why scripps email servers are touching this at all.  Right now, only obvious link is the forwarder to laura's scripps email, which I've temporarily deleted while I debug.  

- [ ]  re-add forwarder to laura

**Comment by andrewsu** [comment]
What I've discovered so far...

- with forwarding restricted to sebastian's and my gmail, emails to help@reframedb.org are now being received. Tested from both my scripps account and Andrea's
- I **do not** receive test emails sent through my gmail client, regardless of whether I'm sending on behalf of my gmail account or scripps account.  I **do** receive them when sending through outlook365.

now will test adding laura's scripps account back to the forwarder...
